skyportal/handlers/api/gcn.py

```diff
@@ -209,6 +218,24 @@ def delete(self, dateobs):
         return self.success()
 
 
+def add_contour(localization_id, request_handler):
+    try:
+        localization = (
+            Localization.query_records_accessible_by(request_handler.current_user)
+            .filter(
+                Localization.id == localization_id,
+            )
+            .first()
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return request_handler.error(f"Unable to generate contour: {e}")
```

Comment:
We cannoy send  responses back to the originating request, which has completed.

Reply:
Ah yes. Now emulating you.